utils/__tests__/redis_storage.test.ts

```diff
@@ -13,20 +13,18 @@ const aRedisKey = "KEY";
 const aRedisValue = "VALUE";
 const aRedisDefaultExpiration = 10;
 
-const setMock = jest
-  .fn()
-  .mockImplementation((_, __, ___, ____, cb) => cb(undefined, "OK"));
-const getMock = jest.fn().mockImplementation((_, cb) => cb(null, aRedisValue));
-const existsMock = jest.fn().mockImplementation((_, cb) => cb(null, 1));
+const setExMock = jest.fn().mockImplementation((_, __, ___) => "OK");
+const getMock = jest.fn().mockImplementation(_ => aRedisValue);
+const existsMock = jest.fn().mockImplementation(_ => 1);
 const redisClientMock = {
   exists: existsMock,
```

Comment:
EXISTS needs to be added because we are using it in `redis_storage.ts` line 140
```ts
  EXISTS: existsMock,
```

Reply:
fixed in https://github.com/pagopa/io-functions-cgn/pull/102/commits/1e5923a1d4661210ca8ea4767ed4016fd7ed2481